app/policies/membership_policy.rb

```diff
@@ -7,7 +7,11 @@ def initialize(person, record)
   end
 
   def index?
-    return true if person.memberships.pluck(:status).include?('member')
+    return true if person.staff == true || person.memberships.pluck(:status).include?('member')
```

Comment:
Not-blocking: It seems like index should always be true, because we scope the the memberships using the resolve below here.